plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -250,16 +258,28 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
             return symbol.dispatchReceiverType ?: symbol.resolvedReceiverTypeRef?.type
         }
 
+    private fun FirPropertySymbol.embedMemberPropertyName() = this.callableId.embedMemberPropertyName(
+        Visibilities.isPrivate(this.visibility)
+    )
+
     /**
      * Construct and register the field embedding for this property's backing field, if any exists.
      */
-    private fun processBackingFields(symbol: FirPropertySymbol, embedding: ClassTypeEmbedding): Pair<SimpleKotlinName, FieldEmbedding>? {
+    @OptIn(SymbolInternals::class)
+    private fun processBackingFields(
```

Comment:
nit: I think the function should have been renamed to `processBackingField` (singular) at some point.